### DISCUSSION TITLE
default secret (kubernetes-dashboard-cert) doesn't have any data , not able to get bearer token

### DISCUSSION THREAD

**Discussion by tejanttf** [kubernetes/dashboard#7154] [Q&A] [answered]
I'm trying to get a barer token, but default secret kubernetes-dashboard-cert doesn't have any data, it is created when applying below yml 

https://raw.githubusercontent.com/kubernetes/dashboard/v2.5.0/aio/deploy/recommended.yaml

kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.5.0/aio/deploy/recommended.yaml
namespace/kubernetes-dashboard created
serviceaccount/kubernetes-dashboard created
service/kubernetes-dashboard created
secret/kubernetes-dashboard-certs created
secret/kubernetes-dashboard-csrf created
secret/kubernetes-dashboard-key-holder created
configmap/kubernetes-dashboard-settings created
role.rbac.authorization.k8s.io/kubernetes-dashboard created
clusterrole.rbac.authorization.k8s.io/kubernetes-dashboard created
rolebinding.rbac.authorization.k8s.io/kubernetes-dashboard created
clusterrolebinding.rbac.authorization.k8s.io/kubernetes-dashboard created
deployment.apps/kubernetes-dashboard created
service/dashboard-metrics-scraper created
deployment.apps/dashboard-metrics-scraper created

kubectl get secret -n kubernetes-dashboard
NAME                              TYPE     DATA   AGE
kubernetes-dashboard-certs        Opaque   0      3s
kubernetes-dashboard-csrf         Opaque   1      3s
kubernetes-dashboard-key-holder   Opaque   0      3s

**Comment by maciaszczykm** [comment, marked as answer]
Check https://github.com/kubernetes/dashboard/tree/master/docs/user/access-control#bearer-token.
